### PR TITLE
Add translations auto update workflow

### PR DESCRIPTION
**Pull request by tobbi**
This makes managing / pulling our translations much easier than before.

It automatically pulls all the translations from Transifex at midnight UTC every Sunday and commits them to the master branch on SuperTux.

### CODE REVIEW

**Review by MatusGuy — APPROVED** [review]
lgtm

File: .github/workflows/translations.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+          # Pull repository:
+          git fetch origin
+          git checkout -b master -t origin/master
+
+          # Pull translations:
+          tx pull --use-git-timestamps -a -f
+
+          # Commit
+          git status
+          git config --global user.email "SuperTux-bot@users.noreply.github.com"
```
Comment (Vankata453) [inline .github/workflows/translations.yml:52]: This is probably not the correct email, as `@users.noreply.github.com` emails consist of a few more numbers in the beginning. Example:

```
78196474+Vankata453@users.noreply.github.com
```

Comment (MatusGuy) [reply .github/workflows/translations.yml:52]: so you're gonna invent a user id for an email? Also, it seems to work

Comment (Vankata453) [reply .github/workflows/translations.yml:52]: > Also, it seems to work

Have there been test pushes done with this email?

Comment (Vankata453) [reply .github/workflows/translations.yml:52]: Per [here](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/setting-your-commit-email-address):

> If you created your account on GitHub.com prior to July 18, 2017, and enabled Keep my email address private prior to that date, your noreply email address from GitHub is [USERNAME@users.noreply.github.com](mailto:USERNAME@users.noreply.github.com).

It *might* be fine, but I don't know the bot account's settings.

Comment (tobbi) [reply .github/workflows/translations.yml:52]: Considering that e-mail is now present, I'm gonna close this discussion

**Review by Zwatotem — APPROVED** [review]
Lgtm

**Pull request merged**